basics/indexing/bloom-filter.md

```diff
@@ -1,22 +1,26 @@
+---
+description: This page describes configuring the bloom filter for Apache Pinot
+---
+
 # Bloom Filter
 
-Bloom filter helps prune segments that do not contain any record matching an **EQUALITY** predicate.
+The bloom filter prunes segments that do not contain any record matching an **EQUALITY** predicate.
 
-It would be useful for a query like the following:
+This is useful for a query like the following:
 
-```
+```sql
 SELECT COUNT(*) 
 FROM baseballStats 
 WHERE playerID = 12345
 ```
 
-There are 3 parameters to configure the Bloom Filter:
+There are 3 parameters to configure the bloom filter:
```

Comment:
Good example of where we can start making the structure more clear and modular.
For each index doc, we can add a procedural header for "Configure the xyz filter" and rewrite explicit steps to complete.
In future, we could regroup similar content into another chunk, perhaps "Configure the table index" where it makes sense to do so

Reply:
1. Set the false positive probability `fpp` for the bloom filter between `0` to `1` (`0.05` by default). As the `fpp` decreases, the bloom filter size and accuracy increases.
...